models/business-unit/src/company/transformers.ts

```diff
@@ -3,16 +3,20 @@ import type { TBusinessUnit, TBusinessUnitGraphql } from '../types';
 
 const transformers = {
   default: Transformer<TBusinessUnit, TBusinessUnit>('default', {
-    buildFields: ['addresses', 'createdBy', 'lastModifiedBy'],
+    buildFields: ['addresses', 'createdBy', 'lastModifiedBy', 'topLevelUnit'],
   }),
   rest: Transformer<TBusinessUnit, TBusinessUnit>('rest', {
-    buildFields: ['addresses', 'createdBy', 'lastModifiedBy'],
+    buildFields: ['addresses', 'createdBy', 'lastModifiedBy', 'topLevelUnit'],
   }),
-
-  //only scaffolding provided at this time
   graphql: Transformer<TBusinessUnit, TBusinessUnitGraphql>('graphql', {
-    buildFields: [],
-    addFields: () => ({
+    buildFields: ['addresses', 'createdBy', 'lastModifiedBy'],
+    replaceFields: ({ fields }) => ({
+      ...fields,
+      topLevelUnit: {
+        ...fields,
```

Comment:
I think this may just need `typeId: 'business-unit'` but I am not yet certain as to why 😿

Reply:
I'm finding some other fields that may be missing from the graphql representation and should be added to the transformer: `parentUnitRef`, `topLevelUnitRef`, `storesRef`.

Reply:
You're completely right, I missed those! I'll add them to the transformer and the tests